results/intfloat__multilingual-e5-small/SNLHierarchicalClustering.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dataset_revision": "3d3d27aa7af8941408cefc3991ada5d12a4273d1",
+  "mteb_dataset_name": "SNLHierarchicalClustering",
+  "mteb_version": "1.6.10",
+  "test": {
+    "evaluation_time": 337.37,
+    "v_measure": 0.6756280728585395,
+    "v_measures": [
+      1.0,
+      0.4103493451179973,
+      0.6244018029078756,
+      0.6676460260153364,
+      0.6757431902514882
+    ]
+  }
```

Comment:
I would redo the results such that you get multiple scores pr. level (using a boostrap similar to fastclustering) and then make it clear which belongs to which level. So make it 

```
"v_measures": {
       "level 1": [1.0, ...],
       "level 2": [0.4103493451179973, ...], ... 
     }

Reply:
Maybe better to allow for the categories to be named?

Reply:
What do you mean by categories?